simtools/corsika/corsika_histograms.py

```diff
@@ -1263,71 +1261,66 @@ def _export_1D_histograms(self, overwrite=False):
                 )
 
     @property
-    def _dict_2D_distributions(self, overwrite=False):
+    def dict_2d_distributions(self):
         """
         Dictionary to label the 2D distributions according to the class methods.
 
-        Parameters
-        ----------
-        overwrite: bool
-            If True overwrites the histograms already saved in the hdf5 file.
-
         Returns
         -------
         dict:
             The dictionary with information about the 2D distributions.
         """
-        if self.__dict_2D_distributions is None:
-            self.__dict_2D_distributions = {
+        if self.__dict_2d_distributions is None:
+            self.__dict_2d_distributions = {
```

Comment:
not sure we want double underscores here. Shouldn't we use a single underscore?

Reply:
I think this was because there was a function "_dict_2d_distribution" and an object "__dist...". The function is now a class function and not private anymore. Change it if you think it is better.

Reply:
Ok, I pushed a commit